as-vending/Dockerfile

```diff
@@ -1,11 +1,24 @@
 # Copyright © 2022 Intel Corporation. All rights reserved.
 # SPDX-License-Identifier: BSD-3-Clause
 
-FROM automated-checkout/build:latest AS builder
+FROM golang:1.18-alpine3.16 AS builder
 
 LABEL license='SPDX-License-Identifier: BSD-3-Clause' \
   copyright='Copyright (c) 2022: Intel'
 
+
+# add git for go modules
+RUN apk update && apk add --no-cache make git gcc libc-dev zeromq-dev linux-headers
+
+ENV GO111MODULE=on
+WORKDIR /usr/local/bin/
+
+# This caches the packages for use when building the other services.
+# Update the go.mod file in this repo when a new package is added to one of the services.
+# This will be obvious when building a service and the un-cached package it loaded every build.
+COPY go.mod .
+RUN go mod download
```

Comment:
You can remove line 26 now that you are doing to go mod download at line 20

Reply:
Good catch @brian-intel